artifacts/definitions/Windows/Timeline/Prefetch.yaml

```diff
@@ -0,0 +1,119 @@
+name: Windows.Timeline.Prefetch
+description: |
+  Windows keeps a cache of prefetch files. When an executable is run,
+  the system records properties about the executable to make it faster
+  to run next time. By parsing this information we are able to
+  determine when binaries are run in the past. On Windows10 we can see
+  the last 8 execution times and creation time (9 potential executions).  
+    
+  This artifact is a timelined output version of the standard Prefetch 
+  artifact. There are several parameter's availible.  
+    - dateAfter enables search for prefetch evidence after this date.  
+    - dateBefore enables search for prefetch evidence before this date.   
+    - binaryRegex enables to filter on binary name, e.g evil.exe.  
+    - hashRegex enables to filter on prefetch hash.     
+
+reference:
+  - https://www.forensicswiki.org/wiki/Prefetch
+
+author: matthew.green@cybereason.com
+
+parameters:
+    - name: prefetchGlobs
+      default: C:\Windows\Prefetch\*.pf
+    - name: dateAfter
+      description: "search for events after this date. YYYY-MM-DDTmm:hh:ssZ"
+      type: timestamp
+    - name: dateBefore
+      description: "search for events before this date. YYYY-MM-DDTmm:hh:ssZ"
+      type: timestamp
+    - name: binaryRegex
+      description: "Regex of executable name."
+    - name: hashRegex
+      description: "Regex of prefetch hash."
+    - name: timeline
```

Comment:
I dont think this is used any more?

Reply:
just removed timeline switches and added the commit